app/models/user.rb

```diff
@@ -19,14 +26,14 @@ def friends
   end
 
   # Users who have yet to confirm friend request
-  def pending_friends
-    friendships.map { |friendship| friendship.friend unless friendship.confirmed }.compact
-  end
+  # def pending_friends
+  #   friendships.map { |friendship| friendship.friend unless friendship.confirmed }.compact
+  # end
```

Comment:
Remove the commented code. Please check for more commented codes especially in your `app/models` directory.